kklib/src/os.c

```diff
@@ -648,7 +648,14 @@ kk_decl_export int kk_os_run_system(kk_string_t cmd, kk_context_t* ctx) {
   }
   #else
   kk_with_string_as_qutf8_borrow(cmd, ccmd, ctx) {
-    exitcode = system(ccmd);
+    int status = system(ccmd);
+    if (WIFEXITED(status)) {
+      exitcode = WEXITSTATUS(status);
+    } else if (WIFSIGNALED(status)) { // TODO: Return differently in this branch
+      exitcode = WTERMSIG(status);
```

Comment:
FWIW haskell returns the negative of this value to distinguish it from an exit status, based on my various failed runs which return `-11` (SIGSEGV) 🤦